official-build.props

```
@@ -0,0 +1,2 @@
+SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-dotnet:1.0.0
```

A .NET build environment should not be necessary for this image.

```suggestion
SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-jdk8:1.0.0
```

---

Done